src/flamegraph/mod.rs

```diff
@@ -305,3 +324,18 @@ fn deannotate(f: &str) -> &str {
     }
     f
 }
+
+impl Default for Direction {
+    fn default() -> Self {
+        Direction::Straight
+    }
+}
+
+impl Direction {
```

Comment:
I don't think `is_inverted` is used anywhere?

Reply:
Removed this method, added Copy,Clone,Eq,PartialEq to Direction.
In `svg.rs` used `opt.direction == Direction::Inverted`